index.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
```

Comment:
HTML structure is good, text has tags, indentation is great. 
Maybe the script elements should go in the head?